config/config_to_options.go

```diff
@@ -0,0 +1,298 @@
+			return options, err
+		}
+
+		// Defaults for safe thread testing
+		c.Db.SQLite.MaxIdleConnections = 1
+		c.Db.SQLite.MaxOpenConnections = 1
+	}
+
+	// Select the datastore
+	if c.Db.Datastore.Engine == datastore.SQLite {
+		tablePrefix := c.Db.Datastore.TablePrefix
+		if len(c.Db.SQLite.TablePrefix) > 0 {
+			tablePrefix = c.Db.SQLite.TablePrefix
+		}
+		options = append(options, engine.WithSQLite(&datastore.SQLiteConfig{
+			CommonConfig: datastore.CommonConfig{
+				Debug:                 c.Db.Datastore.Debug,
+				MaxConnectionIdleTime: c.Db.SQLite.MaxConnectionIdleTime,
+				MaxConnectionTime:     c.Db.SQLite.MaxConnectionTime,
+				MaxIdleConnections:    c.Db.SQLite.MaxIdleConnections,
+				MaxOpenConnections:    c.Db.SQLite.MaxOpenConnections,
+				TablePrefix:           tablePrefix,
+			},
+			DatabasePath: c.Db.SQLite.DatabasePath, // "" for in memory
+			Shared:       c.Db.SQLite.Shared,
+		}))
+	} else if c.Db.Datastore.Engine == datastore.PostgreSQL {
+		tablePrefix := c.Db.Datastore.TablePrefix
+		if len(c.Db.SQL.TablePrefix) > 0 {
+			tablePrefix = c.Db.SQL.TablePrefix
+		}
+
+		options = append(options, engine.WithSQL(c.Db.Datastore.Engine, &datastore.SQLConfig{
+			CommonConfig: datastore.CommonConfig{
+				Debug:                 c.Db.Datastore.Debug,
+				MaxConnectionIdleTime: c.Db.SQL.MaxConnectionIdleTime,
+				MaxConnectionTime:     c.Db.SQL.MaxConnectionTime,
+				MaxIdleConnections:    c.Db.SQL.MaxIdleConnections,
+				MaxOpenConnections:    c.Db.SQL.MaxOpenConnections,
+				TablePrefix:           tablePrefix,
+			},
+			Driver:    c.Db.Datastore.Engine.String(),
+			Host:      c.Db.SQL.Host,
+			Name:      c.Db.SQL.Name,
+			Password:  c.Db.SQL.Password,
+			Port:      c.Db.SQL.Port,
+			TimeZone:  c.Db.SQL.TimeZone,
+			TxTimeout: c.Db.SQL.TxTimeout,
+			User:      c.Db.SQL.User,
+			SslMode:   c.Db.SQL.SslMode,
+		}))
+
+	} else {
+		return nil, spverrors.Newf("unsupported datastore engine: %s", c.Db.Datastore.Engine.String())
+	}
+
+	options = append(options, engine.WithAutoMigrate(engine.BaseModels...))
+
+	return options, nil
+}
+
+func (c *AppConfig) addPaymailOpts(options []engine.ClientOps) []engine.ClientOps {
+	pm := c.Paymail
+	options = append(options, engine.WithPaymailSupport(
+		pm.Domains,
+		pm.DefaultFromPaymail,
+		pm.DomainValidationEnabled,
+		pm.SenderValidationEnabled,
+	))
+	if pm.Beef.enabled() {
+		options = append(options, engine.WithPaymailBeefSupport(pm.Beef.BlockHeadersServiceHeaderValidationURL, pm.Beef.BlockHeadersServiceAuthToken))
+	}
+	if c.ExperimentalFeatures.PikeContactsEnabled {
+		options = append(options, engine.WithPaymailPikeContactSupport())
+	}
+	if c.ExperimentalFeatures.PikePaymentEnabled {
+		options = append(options, engine.WithPaymailPikePaymentSupport())
+	}
+
+	return options
+}
+
+func (c *AppConfig) addTaskManagerOpts(options []engine.ClientOps) []engine.ClientOps {
+	// FIXME: Empty slice declaration using a literal
+	ops := []taskmanager.TasqOps{}
```

Comment:
```suggestion
	// FIXME: Empty slice declaration using a literal
	var ops []taskmanager.TasqOps
```

Reply:
Ahh I forgot to change it, and I put this fixme to make it later when I can run the app 😄